### ISSUE TITLE
Password Reset Email Policy Error - Missing Required Element [Email Address]

### ISSUE THREAD

**Issue by felipecaon** [azure-ad-b2c/samples#406]
Just tried to implement the password reset email policy sample and got Missing Required Element [Email Address] as an error while trying to reset my password.

Did not found the fix for it, looks like the sample has some issue.

https://github.com/azure-ad-b2c/samples/tree/master/policies/pwd-reset-email-exists/policy

**Comment by JasSuri** [comment]
Are you using content definition page layouts version 2 and above?

https://docs.microsoft.com/en-us/azure/active-directory-b2c/page-layout

**Comment by felipecaon** [comment]
Yes I am, just tried to copy and paste the exact same code withour any modifications and got the same error.
